docs/guide/localization/static.md

```diff
@@ -0,0 +1,199 @@
+# Static content localization
+
+This section covers localization of strings throughout template files and Apostrophe user interface. These strings are hard-coded, not editable by logged-in users, and thus are not stored in the database. Instead, this localization is stored in JSON files associated with the various locales.
+
+## Registering strings to localize
```

Comment:
I would just say "localizing strings." "Register" implies that it somehow knows it's registered once you invoke `__t` and will, for instance, populate `en.json` for you, which is not the case. You're invoking the helper and localizing the string right here, you're not registering it for later.

Reply:
I get what you're trying to do with this word, and "localizing" by itself also has the wrong implication since we're just using what you localized in a json file somewhere. How about "displaying localized strings"?